inlong-sort/sort-connectors/hive/src/main/java/org/apache/inlong/sort/hive/filesystem/AbstractStreamingWriter.java

```diff
@@ -195,14 +199,20 @@ public void processWatermark(Watermark mark) throws Exception {
     }
 
     @Override
-    public void processElement(StreamRecord<IN> element) throws Exception {
-        helper.onElement(
-                element.getValue(),
-                getProcessingTimeService().getCurrentProcessingTime(),
-                element.hasTimestamp() ? element.getTimestamp() : null,
-                currentWatermark);
-        if (metricData != null) {
-            metricData.invokeWithEstimate(element.getValue());
+    public void processElement(StreamRecord<IN> element) {
+        try {
+            helper.onElement(
+                    element.getValue(),
+                    getProcessingTimeService().getCurrentProcessingTime(),
+                    element.hasTimestamp() ? element.getTimestamp() : null,
+                    currentWatermark);
+            if (metricData != null) {
+                metricData.invokeWithEstimate(element.getValue());
+            }
+        } catch (Exception e) {
```

Comment:
Is it necessary to print the exception info in the log?

Reply:
I will improve in the next pr, it should support dirty data side-output.